kolibri/plugins/learn/templates/learn/learn.html

```diff
@@ -1,11 +1,20 @@
-{% extends "kolibri/base.html" %}
```

Comment:
I was actually trying to figure out how to simplify the template. Right now for example we have `base_frontend_async`, `async_frontend_assets`, and `content_renderer_frontend_async` which all trickle down to `render_as_async`. Similar situation for sync stuff.

I was trying to figure out if we could just have two customs tags like `async_assets` and `sync_assets` (passing in arguments using `takes_context=True`) to handle all the various cases at once depending on what hooks are passed in.